libs/ui/src/hooks/use-activity.hook.ts

```diff
@@ -52,15 +52,25 @@ const transformApiData = (
       activityData.transactionStatus = TransactionStatusEnum.applied;
     }
 
+    const getTokenSymbol = (address: string) => {
+      if (userTokensMetadata[address].hasOwnProperty('optimized_symbol')) {
+        return userTokensMetadata[address].optimized_symbol;
+      }
+
+      return userTokensMetadata[address].name;
```

Comment:
could we simplify it to 
`const getTokenSymbol = (address: string) => userTokensMetadata[address].optimized_symbol ??userTokensMetadata[address].name` ?